packages/assistify-ai/server/methods/SmartiWidgetBackend.js

```diff
@@ -156,6 +156,13 @@ Meteor.methods({
 		};
 	},
 
+	closeConversation(roomId) {
+		if (RocketChat.authz.hasAllPermission(Meteor.userId(), 'archive-room', roomId)) { return false ; }
+		const room = RocketChat.models.Rooms.findOneById(roomId);
+		if (room.archived) { return false; }
+		return true;
```

Comment:
```suggestion

```

Reply:
seeing this line, `return !room.archived;` really makes sense ;)

Reply:
Wasn't possible to have only one single comment to two code lines... already gave feedback for that.